tests/gold_tests/remap/remap_https.test.py

```diff
@@ -59,7 +58,7 @@
     'map https://www.example.com:{1} http://127.0.0.1:{0}'.format(server.Variables.Port, ts.Variables.ssl_port)
 )
 ts.Disk.remap_config.AddLine(
-    'map https://www.anotherexample.com https://127.0.0.1:{0}'.format(server2.Variables.SSL_Port,ts.Variables.ssl_port)
+    'map https://www.anotherexample.com https://127.0.0.1:{0}'.format(server2.Variables.SSL_Port, ts.Variables.ssl_port)
 )
```

Comment:
Didn't work before either.  But the www.anotherexample.com probably needs a :{1} to pick up the ts.Variables.ssl_port